auth/auth_test.go

```diff
@@ -7,13 +7,14 @@ import (
 	"net/http"
 	"net/http/httptest"
 	"testing"
+	"time"
 )
 
 type Handler func(w http.ResponseWriter, r *http.Request)
 
 func newAuth0ManagerAndServer(handler Handler) (*Auth0Manager, *httptest.Server) {
 	server := httptest.NewServer(http.HandlerFunc(handler))
-	auth0Manager := NewAuth0Manager(server.URL, "", "", 500)
+	auth0Manager := NewAuth0Manager(server.URL, "", "", 500*time.Millisecond)
```

Comment:
Better format when specifying some sort of rate is `period/events`, so this would be `time.Second/2`